composer.json

```diff
@@ -1,17 +1,27 @@
 {
-	"name": "friendsoftypo3/migrations",
-	"type": "typo3-cms-extension",
-	"description": "Enable SQL migration powered by Doctrine",
-	"license": ["GPL-2.0-or-later"],
-	"keywords": ["TYPO3 CMS", "Migration"],
-	"require": {
-		"php": "^7.4",
-		"doctrine/migrations": "^3.0",
-		"typo3/cms-core": "^10.0"
-	},
-	"autoload": {
-		"psr-4": {
-			"FriendsOfTYPO3\\Migrations\\": "Classes/"
-		}
-	}
+  "name": "friendsoftypo3/migrations",
```

Comment:
Not sure if we should migrate the namespace to visol. If we want it to be in friendsoftypo3, we need to discuss this with the actual friends of T3 (criteria for adding the EXT etc.)

I already asked Benni about this (last year) and got the following reply:

>  prinzipiell denkbar, was wir dafür aber brauchen ist eine Gruppe von Maintainern die da hinterher sind und auch dran bleiben (min 1-2 Jahre), da ansonsten das Projekt verwaist und ich letztendlich das ausbaden muss. Also wenn das gegeben ist (und evtl. etwas mehr Doku) wäre das schon sicherlich cool